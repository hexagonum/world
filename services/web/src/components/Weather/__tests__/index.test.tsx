import { render } from '@testing-library/react';
import { Weather } from '..';

describe('Weather', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Weather city="" latitude={0} longitude={0} timezone={0} />
    );
    expect(container).toMatchSnapshot();
  });
});
