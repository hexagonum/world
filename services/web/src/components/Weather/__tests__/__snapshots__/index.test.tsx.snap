// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather to match snapshot 1`] = `
<div>
  <div
    class="chakra-card border border-gray-200 css-2dlz7"
  >
    <div
      class="chakra-card__body css-0"
    >
      <div
        class="flex justify-center items-center"
      >
        fetch is not defined
      </div>
    </div>
  </div>
</div>
`;
